src/components/AccountMenuDropdown.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react'
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group'
+import '../styles/account_menu_dropdown.css'
+
+class AccountMenuDropdown extends React.Component {
+  constructor(props) {
+    super(props);
+    this.toggleMenu = this.toggleMenu.bind(this);
+    this.state = {
+      menuActive: false
+    };
+  }
+
+  toggleMenu() {
+    let menuState = !this.state.menuActive;
+    this.setState({
+      menuActive: menuState
+    });
+  }
+
+  render() {
+    let menu;
```

Comment:
does this have to be ```let``` or can it be ```const```?

And also Jrob likes whitespace between the set up and body of a render